libgraphics/include/pivot/graphics/types/Transform.hxx

```diff
@@ -38,5 +61,23 @@ public:
 
     /// Scale component
     glm::vec3 scale = glm::vec3(1.0f);
+
+    /// Root of the transform
+    pivot::EntityRef root = EntityRef::empty();
 };
+
+/// Ostream overload for glm::vec3
+inline std::ostream &operator<<(std::ostream &os, const glm::vec3 &vec)
+{
+    return os << "(" << vec.x << "," << vec.y << "," << vec.z << ")";
+}
+
+/// Ostream overload for Transform
+inline std::ostream &operator<<(std::ostream &os, const Transform &transform)
+{
+    return os << "Position: " << transform.position << std::endl
+              << "Rotation: " << transform.rotation << std::endl
+              << "Scale: " << transform.scale << std::endl
+              << "Root: " << transform.root;
```

Comment:
`srd::endl` in a ostream overload 😬

Reply:
Not really any way to display this nicely otherwise